### PR TITLE
feat: Support context menu for text operations - support it for inspect element

### PR DESCRIPTION
**Pull request by iongion**
Signed-off-by: Ionut Stoica <stoica.ionut@gmail.com>

# Added

* Added `electron-context-menu` - it brings right click support of electron that allows clipboard operations among this, which is always useful. It also bring `Inspect element` support during development, like web developers expect.

PS: Can't take a screenshot with it on Linux as in the year 2022, we can't take screenshots of native context menus in Linux graphical servers. Neither dropdown boxes!

### CODE REVIEW

File: packages/main/src/mainWindow.ts [hunk 1]
```diff
@@ -105,6 +106,10 @@ async function createWindow() {
     browserWindow.destroy();
   });
 
+  contextMenu({
+    showInspectElement: import.meta.env.DEV,
+  });
+
```
Comment (benoitf) [inline packages/main/src/mainWindow.ts]: For developer mode I think it make sense for the inspect even if the developer console is always there but then for production it brings other menus that probably should not be displayed
image



```suggestion
  if (import.meta.env.DEV) {
  contextMenu({
    showInspectElement: true,
  });
}
```

Comment (iongion) [reply packages/main/src/mainWindow.ts]: There are options you do want to keep for production too, let's decide, these you do want

```
cut
copy
paste
```

https://github.com/sindresorhus/electron-context-menu#readme

**Review by benoitf — APPROVED** [review]
I see only `Copy` now

image

**Comment by iongion** [comment]
Cool, the cut-paste will be visible for text input fields / text areas. There is still suggest of correct word if you misspell. But that can be disabled using the `webConfig` flag. It is useful in my opinion if you leave spelling on (as it is by default)!

**Pull request merged**